GOVERNANCE.rst

```diff
@@ -384,24 +431,32 @@ If the proposed change cannot gather a 2/3 majority of the votes cast,
 the CC tries to resolve the deadlock.
 
 -  The CC will use consensus seeking.
--  If no consensus can be found within a month, the decision is made
-   through a simple majority vote (with tie breaking) among the CC
+-  If no consensus can be found within twenty working days excluding week-ends
+   since the beginning of the stage-1 "reasonable time for consideration" period,
+   the decision is made through a simple majority vote (with tie breaking) among the CC
    members.
--  Any CC decision must be supported by an :ref:`steps`, which has been made public and discussed before the vote.
+-  If this vote is needed, the vote should take place within twenty-five working days after the beginning
+   of the stage-1 "reasonable time for consideration" period, during a regular, minuted CC meeting.
+-  CC decisions on major changes must be supported by an :ref:`steps`,
+   which has been made public and discussed before the vote.
 
 .. _steps:
 
 sktime enhancement proposal
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-For all decision of the CC, an sktime enhancement proposal (STEP) must have been made
+For all stage 3 decisions of the CC on major changes,
+an sktime enhancement proposal (STEP) must have been made
 public and discussed before the vote.
 
 An sktime enhancement proposal is a consolidated document, with a concise
 problem statement, a clear description of the proposed solution and a
 comparison with alternative solutions, as outlined in our
 `template <https://github.com/sktime/enhancement-proposals/blob/master/TEMPLATE.md>`__.
 
+A complete STEP must always include at least a high-level design for the proposed change,
```

Comment:
I think more info on what consistute a (good) STEP should go into the README of the enhancement-proposal repo.

Reply:
yes, but that's additional work, right? Should we open an issue?